java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -272,6 +272,32 @@ CompletableFuture<Double> zaddIncr(
      */
     CompletableFuture<Map<String, Double>> zpopmax(String key);
 
+    /**
+     * Blocks the connection until removes and returns a member with the highest score from the sorted
+     * sets stored at the specified <code>keys</code>.<br>
+     * <code>BZPOPMAX</code> is the blocking variant of {@link #zpopmax(String)}.<br>
+     * A member with the highest score is popped from first sorted set that is non-empty, with the
+     * given <code>keys</code> being checked in the order that they are given.
+     *
+     * @see <a href="https://redis.io/commands/bzpopmax/">redis.io</a> for more details.
+     * @apiNote <code>BZPOPMAX</code> is a client blocking command, see <a
+     *     href="https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands">Blocking
+     *     Commands</a> for more details and best practices.
+     * @param keys The keys of the sorted sets.
+     * @param timeout The number of seconds to wait for a blocking <code>BZPOPMAX</code> operation to
```

Comment:
```suggestion
     * @param timeout The number of seconds to wait for a blocking operation to
```
Nit: Don't think you need to specify the command name seems trivial.